packages/rsocket-core/src/__tests__/RSocketResumableTransport-test.js

```diff
@@ -309,11 +305,6 @@ describe('RSocketResumableTransport', () => {
       currentTransport.mock.connect();
     });
 
-    it('on transport CONNECTING it updates to NOT_CONNECTED state', () => {
-      currentTransport.mock.connecting();
-      expect(resumableStatus.kind).toBe('NOT_CONNECTED');
-    });
-
```

Comment:
This was removed as this state is not possible: underlying connection cant go CONNECTED -> CONNECTING by itself, but CONNECTED -> CLOSED | ERROR. Then `ResumableTransport` creates fresh connection once requested externally with `connect` function